exawind/Pazy_Wing/README.md

```diff
@@ -0,0 +1,177 @@
+# Pazy Wing Aeroelastic Validation Case
+
+This benchmark contains a blade-resolved model of the Pazy Wing [1] aeroelastic case in uniform flow.  A range of freestream velocities is used to assess the pre-flutter static deflection and flutter onset boundaries.  Both wind-tunnel data [1] and simulation results [2] are used for comparison.
+
+**Contents**
+
+- [Simulation Description and Setup](#simulation-description-and-setup)
+- [Postprocessing](#postprocessing)
+- [Code Performance](#code-performance)
+- [Results](#results)
+
+## Simulation Description and Setup
+
+The geometry and flow conditions of the Pazy Wing case are defined in Avin et al. [1]  They are summarized in the following table.
+
+| Case Property     | Value                  |
+| :---              | ---                    |
+| Span              | 0.55 m                 |
+| Chord             | 0.10 m                 |
+| Airfoil           | NACA0018               |
+| Angle of Attack   | 3-7 deg                |
+| Density           | 1.225 kg/m<sup>3</sup> |
+| Dynamic Viscosity | 1.8E-05 Pa-s           |
+| Velocity          | 15-50 m/s              |
+| Mach Number       | 0.04-0.15              |
+| Reynolds Number   | < 3.4E+05              |
+| Turbulence Level  | 0.5%                   |
+
+Note that the wind-tunnel model given in [1] has several small unintentional features, including varying airfoil thickness along the span, sag in the flap-wise direction, and twist (about 1 degree).  It is assumed here that these features have a negligible impact on the FSI results, and an idealized version of the wing is used instead, with a uniform cross-section along the span.
```

Comment:
It might be worth restating it so that the nominal blade model is a 2D airfoil cross-section with no spanwise variations, and that this "idealized version" of the wing is the same as the "as designed/as intended" version of the wing, unless I'm misunderstanding how it was presented in the study.  

Do we know if other simulations in the study used the idealized model, or did they used the as-built model?

Reply:
I re-worded this paragraph to make this more clear.

It is unclear to me whether the workshop participants used the as-built or idealized versions.  At some point, I'd like to add another set of exawind results for my best stab at the as-built geometry.  (The full details of the as-built are not really given in the original paper.  There are some very awkward 3D line plots trying to show what they measured the spanwise variation to be, but there are very few data points, and they are hard to read.)

Reply:
Thanks Neil!  Yeah, we can additional results later that show what the geometry differences do to the results.